tests/e2e/config.json

```diff
@@ -0,0 +1,8 @@
+{
+        "image_url": ["nginx", "nginx"],
+        "k8smasterforkubeedge":"https://172.17.0.2:6443",
+        "dockerhubusername":"user",
+        "dockerhubpassword":"password",
+        "mqttendpoint":"tcp://127.0.0.1:1884",
+        "kubeconfigpath":"/home/ubuntu/.kube/config"
```

Comment:
Add a new file?

Reply:
It's an extra file created during local tests and it's deleted.